docs/Building.md

```diff
@@ -4,47 +4,66 @@
+1. In the "Welcome to IntelliJ IDEA" window, click "Open or import" and open the project's root directory. (If you do not see the welcome window, File > Open can do the same.)
 
-If you make changes to most IntelliJ settings, they will save over the files that are checked into Git. These changes can be shared by committing the updated files.
+1. In the Gradle tool window (expand by clicking the Gradle button on right top), click ![the Reload All Gradle Projects button](https://resources.jetbrains.com/help/img/idea/2020.2/icons.actions.refresh.svg) on the toolbar.
 
-There are a few settings that we use for consistency that are saved in the user-specific `workspace.xml` file. These settings must be enabled manually by each developer:
+1. Build > Build Project.
 
-* Preferences > Editor > General > Auto Import > Optimize imports on the fly (for current project)
+1. Preferences > Editor > General > Auto Import > Optimize imports on the fly (for current project)
 
-* (optional, but useful) Preferences > Build, Execution, and Deployment > Build Tools > Maven > Importing > Automatically download > Sources
+1. *(This is optional, but useful.)*\
+ Preferences > Build, Execution, and Deployment > Build Tools > Maven > Importing > Automatically download > Sources
 
-Before attempting to build anything, exit IntelliJ and build once from the command line. This will generate the protobuf files.
+**Now you have configured IntelliJ successfully!**
 
-```
-./gradlew package
-```
+*(Skip if you are not interested in running the tests.)*\
+To verify that you have a working configuration, pick some tests that depend on FDB (such as those in the `FDBRecordStoreQueryTest` fixture), and check you can run them from IntelliJ.
 
-Launch IntelliJ again and Build > Build Project should succeed.
+If this does not work smoothly for you, please create a [new issue](https://github.com/FoundationDB/fdb-record-layer/issues/new) or ask in [the forum](https://forums.foundationdb.org/c/using-layers).
 
-To verify that you have a working configuration, pick some tests that depend on FDB (such as those in the `FDBRecordStoreQueryTest` fixture), and check that you can run them from IntelliJ.
+### Behind the Scene
 
-## Running the tests
+*(This section explains what happened above and why, in case you are interested.)*
 
-In order to run the tests you will need to have FDB installed on your development machine. The latest FDB binaries can be found on
-the [FoundationDB website](https://www.foundationdb.org/download/).
+The `./gradlew package` command (Step 2) generates the protobuf files.
+
+If you are used to opening older `.ipr`-based IntelliJ projects, the process for opening this project will be somewhat different. Instead of selecting the `fdb-record-layer.ipr` file, open the project's root directory; IntelliJ will find the project configuration in the `.idea` directory.
```

Comment:
This doesn't really feel like it belongs here any more.  Maybe it should just be deleted and we forsake those using old IntelliJ versions?  If anyone things it should belong here, it should probably be included in the previous section that describes how to open the project.

Reply:
Deleted.